tensorboard/webapp/BUILD

```diff
@@ -156,3 +147,27 @@ tf_svg_bundle(
     ],
     out = "icon_bundle.svg",
 )
+
+tf_sass_library(
+    name = "angular_material_theming",
+    srcs = [
+        "_angular_material_theming.scss",
+        "@npm//:node_modules/@angular/material/_theming.scss",
+    ],
+)
+
+tf_sass_library(
+    name = "theme",
+    srcs = [
+        "_palette.scss",
+        "_theme.scss",
+    ],
+    deps = [
+        ":angular_material_theming",
+    ],
+)
+
+tf_sass_binary(
+    name = "styles",
+    src = "styles.scss",
```

Comment:
I wonder whether this should depend on something.

Reply:
Yes but kinda hidden. While working internally, Erik and I realized every sass_binary requires dependencies on 'webapp/theme' so we decided to provide by that default in the defs.bzl (you cannot specify the same dependency twice; bazel will yell at you).